packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -298,8 +300,33 @@ const withChunking = (config) => {
     }
 }
 
+// TODO: Once we create a new project for extensibility we'll have the opportunity to better move this utility
+// to a place that can be reused. This util is also used in the `extensions-loader`.
+const normalizeExtensionsList = (extensions = []) =>
+    extensions.map((extension) => {
+        return {
+            packageName: Array.isArray(extension) ? extension[0] : extension,
+            config: Array.isArray(extension) ? {enabled: true, ...extension[1]} : {enabled: true}
+        }
+    })
+
 const staticFolderCopyPlugin = new CopyPlugin({
-    patterns: [{from: 'app/static/', to: 'static/'}]
+    patterns: [
+        {
+            from: 'app/static/',
+            to: 'static/'
+        },
+        ...normalizeExtensionsList(appConfig?.extensions).map((extension) => {
+            const {packageName} = extension
+            // Parse the extension name out.
+            return {
+                from: `${projectDir}/node_modules/${packageName}/static`,
+                to: `static/${EXTENIONS_NAMESPACE}/${packageName}`,
+                // Add exclude for readme file.
+                noErrorOnMissing: true
+            }
+        })
+    ]
```

Comment:
nit, what's your thought on making a new instance of the `CopyPlugin` that is separate from the existing plugin? that will make it more portable in case we need to move it.

Reply:
like 
```js
const staticFolderCopyPlugin = new CopyPlugin({
    patterns: [
        {
            from: 'app/static/',
            to: 'static/'
        }]
})

const extensionStaticCopyPlugin = new CopyPlugin({
   ......normalizeExtensionsList(appConfig?.extensions).map(...)
})
```

Reply:
Yeah.. so that is going to move for sure when we create the new package. I don't think it's gonna be much of a big deal to do that later during the move itself. That was something that also went through my head.

Reply:
What we'll most likely do is create something like `pwa-kit-extensibility/src/webpack/plugins/copy-extension-assets` and we would import and instantiate that in the webpack config in the dev project.